ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/IdeaFrame.kt

```diff
@@ -56,10 +59,52 @@ class IdeaFrame(remoteRobot: RemoteRobot, remoteComponent: RemoteComponent) : Co
 
     fun openProjectStructure() = step("Open Project Structure dialog") {
         if (remoteRobot.isMac()) {
-            keyboard { this.hotKey(KeyEvent.VK_META, KeyEvent.VK_SEMICOLON) }
+            keyboard { hotKey(KeyEvent.VK_META, KeyEvent.VK_SEMICOLON) }
         } else {
-            keyboard { this.hotKey(KeyEvent.VK_SHIFT, KeyEvent.VK_SHIFT, KeyEvent.VK_ALT, KeyEvent.VK_S) }
+            keyboard { hotKey(KeyEvent.VK_SHIFT, KeyEvent.VK_SHIFT, KeyEvent.VK_ALT, KeyEvent.VK_S) }
         }
         find(ComponentFixture::class.java, byXpath("//div[@accessiblename='Project Structure']")).click()
     }
+
+    // Show AWS Explorer, or leave it open if it is already open
+    fun showAwsExplorer() {
+        try {
+            find<AwsExplorer>(byXpath("//div[@class='ExplorerToolWindow']"))
+        } catch (e: Exception) {
+            find(ComponentFixture::class.java, byXpath("//div[@accessiblename='AWS Explorer' and @class='StripeButton' and @text='AWS Explorer']")).click()
+        }
+    }
+
+    fun setCredentials(profile: String, region: String) {
```

Comment:
I would name this to indicate it uses the status bar since we have a new way to do it soon

Reply:
It's part of the the idea frame instead of the explorer frame so I think it's evident from that

Reply:
Fair enough